main.py

```diff
@@ -1,19 +1,24 @@
 from data import collect_matrix, output_matrices, check_matrices
 from multiplication import multiplicate
+from generation import generate_matrices
 
-print("-- ENTER THE FIRST MATRIX --")
-matrix1 = collect_matrix()
+question = int(input("select a mode:\n1 - your matrices\n2 - generate random matrices\n! enter only a number !\n"))
```

Comment:
use [argparse](https://docs.python.org/3/howto/argparse.html) python module to handle input from command line. 
propose list of input parameters such as : mode , matrices size , etc